stripe/src/main/java/com/stripe/android/paymentsheet/repositories/PaymentMethodsRepository.kt

```diff
@@ -65,8 +83,39 @@ internal sealed class PaymentMethodsRepository {
             }.getOrDefault(emptyList())
         }
 
-        private companion object {
-            private val PRODUCT_USAGE = setOf("PaymentSheet")
+        override suspend fun save(
+            customerConfig: PaymentSheet.CustomerConfiguration,
+            paymentMethodCreateParams: PaymentMethodCreateParams
+        ): PaymentMethod = withContext(workContext) {
+            runCatching {
+                stripeRepository.createPaymentMethod(
+                    paymentMethodCreateParams,
+                    ApiRequest.Options(
+                        publishableKey,
+                        stripeAccountId
+                    )
```

Comment:
I would wrap `stripeRepository.createPaymentMethod()` in `requireNotNull`. We should aggressively throw if we get a null response.

e.g.
```
requireNotNull(...) {
    "Could not parse PaymentMethod."
}
```